iocage_lib/ioc_start.py

```diff
@@ -434,12 +434,17 @@ def __start_jail__(self):
                         # Jails default is epairNb
                         interface = f"{interface.replace('vnet', 'epair')}b"
 
-                    cmd = ["jexec", f"ioc-{self.uuid}", "ifconfig", "-f",
-                           "inet:cidr",
+                    cmd = ["jexec", f"ioc-{self.uuid}", "ifconfig",
                            interface, "inet"]
                     out = su.check_output(cmd)
 
-                    addr = f"{out.splitlines()[2].split()[1].decode()}"
+                    hexmask = out.splitlines()[2].split()[3].decode()
```

Comment:
Just noticed the duplication here. Let's make out.splitlines().... A variable called addr_split and have hexmask and ip4_addr use that by index with their decode. Saves us an operation then.